smartfs.lua

```diff
@@ -1046,9 +1111,9 @@ smartfs.element("code", {
 
 		return self.data.code
 	end,
-	submit = function(self, fields, player)
```

Comment:
Doesn't this break code? ie: you need to be able to get the whole fields table, as you may add more than one element